picoGPT/app.py

```
@@ -0,0 +1,33 @@
+# app.py
+
+from calculator import calculate
+from constants import ADD, SUBTRACT, MULTIPLY, DIVIDE, ERROR_INVALID_OPERATION
```

Ruff (0.8.2)

`constants.ADD` imported but unused

Remove unused import

(F401)